org.idempiere.webservices/WEB-INF/src/org/idempiere/adinterface/ModelADServiceImpl.java

```diff
@@ -1213,6 +1209,9 @@ public StandardResponseDocument scanFields(DataField[] fields,MWebServiceType m_
 								+ ": input column " + field.getColumn() + " does not exist");
 					} else {
 						try {
+							if(!fieldInput.getAD_Column().isAlwaysUpdateable() && isProcessed)

```

Comment:
fieldInput.getAD_Column() is inefficient, should use MColumn.get(fieldInput.getAD_Column_ID()) instead.